config/deploy.rb

```diff
@@ -15,6 +15,8 @@
 
 set :branch, 'master'
 
+set :passenger_restart_with_touch, true
```

Comment:
Do I leave the "after :publishing, :restart" statement or does that happen magically as well?